src/Debug.h

```diff
@@ -132,6 +135,11 @@ void ResyncDebugLog(FILE *newDestination);
         } \
    } while (/*CONSTCOND*/ 0)
 
+
```

Comment:
I like seeing double empty lines between major code blocks, but the original code does not use them here, and Amos does not like seeing them, so let's not add this one.

Reply:
It's not just for me - the code validation tools flag it up as a problem indicative of bad coding practice and astyle strips it out forcing post-commit changes by the maintenance systems.

Reply:
One more reason we should not fight this particular wrong IMO.